src/elm/Favorites.elm

```diff
@@ -93,6 +93,25 @@ updateFavorites user favorite =
             ( [], False )
 
 
+{-| enableFavorite : takes current user and favorite key and adds favorite to list of favorites
+-}
+enableFavorite : WebData CurrentUser -> String -> ( List String, Bool )
+enableFavorite user favorite =
```

Comment:
thoughts on making this `addFavorite`?

```suggestion
addFavorite : WebData CurrentUser -> String -> ( List String, Bool )
addFavorite user favorite =
```

Reply:
yeah for sure — I wasn't sure about the naming norms regarding `AddFavorite` and `addFavorite` since there was `ToggleFavorite` and `updateFavorite`, but I think `addFavorite` definitely makes more sense